chi/container.py

```diff
@@ -76,9 +77,14 @@ def create_container(
             such as the "nvidia" runtime provided by NVIDIA Jetson Nano/TX2.
         start (bool): Whether to automatically start the container after it
             is created. Default True.
+        wrapped_call (bool): Whether the function was called from within
```

Comment:
I have some concerns about `wrapped_call`. My reading is that by default now, `create_container` will instead `ensure_container` exists. Instead, I think we should leave `create_container` alone to preserve legacy behavior for users and Trovi artifacts. No changes should be made in `create_container`.

Reply:
Is there some reason you decided `wrapped_call` was needed?

Reply:
Totally an oversight on my part–I thought that `ensure_container` might end up being recursive since it calls `create_container` and is also called inside `create_container`, but I realize that this is not the case.